client/windows/CHeroBackpackWindow.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * CHeroBackpackWindow.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#include "StdInc.h"
+#include "CHeroBackpackWindow.h"
+
+#include "../gui/CGuiHandler.h"
+
+CHeroBackpackWindow::CHeroBackpackWindow(const CGHeroInstance * hero)
```

Comment:
While functionality is definitely something that players requested, but what about visuals?
Are you planning to make them relatively soon, e.g. in week or two?
- If you can make them in time for 1.3 then I am OK with accepting this PR as it
- Alternatively, I can look into this myself when I can find time.

In later case would be great to have backpack disabled by default (macro or hidden setting) until backpack will be made presentable to players.

Reply:
> but what about visuals?

I am not familiar with make visuals. I'm not sure if I can do it right.

> until backpack will be made presentable to players.

Since the button to open the backpack doesn't look presentable and needs to be redesigned, I planned to remove it altogether before the merge.